docs/root/development/testing/local_stats.rst

```diff
@@ -0,0 +1,40 @@
+.. _testing_local_stats:
+
+Local Stats
+===========
+
+The `local-stats` `branch <https://github.com/lyft/envoy-mobile/tree/local-stats>`_ allows a
+developer to run a local statsd server, and see stats emissions from a client running in the
+simulator/emulator.
```

Comment:
suggestion(optional, but it's probably useful, since it wasn't obvious for me) to emphasize that this setup/branch will not work with real device.

```suggestion
simulator/emulator. Note: this setup will not work with a physical device.

Reply:
I thought that saying a "client running in the simulator/emulator" was explicit enough, but happy to add the note.

https://github.com/lyft/envoy-mobile/pull/1081